.php_cs.dist

```diff
@@ -54,6 +54,7 @@ $config = PhpCsFixer\Config::create()
         'single_line_comment_style' => true,
         'strict_comparison' => true,
         'strict_param' => true,
+        'string_line_ending' => true,
```

Comment:
Should be after `single_line_comment_style`.

Reply:
why?

Reply:
can you give an example for that?

Reply:
Because I was tired and read `strict_line_ending` :D ... Nothing to do here.

Reply:
:D  thanks for reviewing